docs/licensing.asciidoc

```diff
@@ -51,24 +49,20 @@ data:
 <1> this label is required for ECK to identify your license secret
```

Comment:
This existed before my change, but I think the numbers are because they are like footnotes - so the numbers help you map which item in the example corresponds to which footnote